src/main/java/App/Main.java

```diff
@@ -0,0 +1,32 @@
+package App;
+import Entity.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.net.URL;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        URL url = new URL("https://www.metaweather.com/api/location/638242");
```

Comment:
What if I do not know the woeid of a city I need weather? You may need to resolve the city to woeid first and then call for weather as you are already doing.